centrifuge-app/src/components/PoolCard/index.tsx

```diff
@@ -76,42 +110,82 @@ export function PoolCard({
   metaData,
   createdAt,
 }: PoolCardProps) {
+  const theme = useTheme()
   const isOneTranche = tranches && tranches?.length === 1
-  const renderText = (text: string) => (
-    <Text fontWeight={500} as="h2" variant="body1">
-      {text}
-    </Text>
-  )
+  const isTinlakePool =
```

Comment:
```suggestion
  const isTinlakePool = poolId.startsWith('0x')
```

this should be enough since only tinlake pools have hex coded IDs